core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -375,47 +375,46 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 		return errors.Wrap(err, "building transmit checker")
 	}
 
+	lgr, err := etx.GetLogger(eb.logger.With(
+		"gasPrice", attempt.GasPrice,
+		"gasTipCap", attempt.GasTipCap,
+		"gasFeeCap", attempt.GasFeeCap,
+	))
+	if err != nil {
+		eb.logger.Errorw("failed to get tx logger meta of the transaction", "err", err)
+	}
```

Comment:
In the case where `err != nil`, would `lgr` be nil? Should you do `lgr = eb.logger.With(...)` in this case as a fall back?